Issue #32: Replaced annotations with attributes.

alexmerlin opened the pull request.

alexmerlin committed:
Issue #32: Replaced annotations with attributes.

Signed-off-by: alexmerlin <alex.merlin.1985@gmail.com>

alexmerlin committed:
cover missing lines

Signed-off-by: alexmerlin <alex.merlin.1985@gmail.com>

alexmerlin committed:
cover missing lines

Signed-off-by: alexmerlin <alex.merlin.1985@gmail.com>

alexmerlin committed:
cover missing lines

Signed-off-by: alexmerlin <alex.merlin.1985@gmail.com>

alexmerlin committed:
Updated readme

Signed-off-by: alexmerlin <alex.merlin.1985@gmail.com>

alexmerlin committed:
Added psr/container to compoer.json

Signed-off-by: alexmerlin <alex.merlin.1985@gmail.com>

arhimede reviewed (changes requested):
remove php 8.1
list with bullets to readme.md any major differences between 4.x and 5.x

alexmerlin committed:
Removed support for PHP 8.1
Updated readme.md with the most important changes since version 5.0
Allowed psr/container version 1.x
Added security.md

Signed-off-by: alexmerlin <alex.merlin.1985@gmail.com>
